.template/addons/github/.github/workflows/automated_code_review.yml.tt

```diff
@@ -0,0 +1,59 @@
+name: Automated code review
+
+on:
+  pull_request:
+
+env:
+  DANGER_GITHUB_API_TOKEN: ${{ github.token }}
+
+jobs:
+  automated_code_review:
+    name: Run Danger
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      <%- if WEB_VARIANT -%>
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '<%= NODE_VERSION %>'
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Yarn cache
+        uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: node_modules cache
+        id: node-modules-cache
+        uses: actions/cache@v2
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-nodemodules-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-nodemodules-
+
+      - name: Yarn install
+        run: yarn
+        if: |
```

Comment:
Shouldn't the conditional be placed above the `run` command?

Reply:
Yes, fixed in d69760c.